frontend/integration-tests-cypress/cypress/tests/monitoring.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { timeouts } from '../support/utils/timeouts';
+import { testName } from '../support/utils/common';
+import { submitButton, errorMessage } from '../support/utils/selectors';
+import { listPage } from '../support/views/list-page';
+import { detailsPage } from '../support/views/details-page';
+
+const isWatchdogAlertDetailsPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alert Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+const isWatchdogAlertRulesPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alerting Rule Details');
+  detailsPage.sectionHeaderShouldExist('Active Alerts');
+};
+
+const isWatchdogSilencePage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Silence Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+before(() => {
+  cy.createTestProject(testName);
+});
+after(() => {
+  cy.deleteTestProject(testName);
+});
+
+describe('Monitoring: Alerts', () => {
+  // each it() should be its own atomic test (run independently of other tests), per best practices.
```

Comment:
Just realized there is something called an alias that seems interesting:

```typescript 
  beforeEach(() => {
    cy
        .get('a#some-id')
        .as('button') // @button becomes an alias. can create several alias as test setup.
  }) 
  it('Does a thing with an aliased button', function() {
    cy
        .get('@button')  // use alias in specific test
        .should('be.visible')
   });
```
Not sure if its actionable right now, but seems like it could be a handy tool.

Reply:
I guess anything can be an alias:

```typescript
cy.get('a#some-id').as('button) // @button 
cy.request('http://foobar.com').as('request') // @request, for an http request
cy.wait('@request') // wait for request
```

Reply:
yes, saw that early on, it's another level of abstraction.  We could probably use for helpers like [this](frontend/integration-tests-cypress/cypress/support/utils/selectors.ts):
```
export const submitButton = 'button[type=submit]';

import { submitButton } from '../support/utils/selectors';
cy.get(submitButton).click();
```
..to..
```
cy.get('button[type=submit]').as('submitButton');
cy.get('.pf-c-alert.pf-m-inline.pf-m-danger').as(errorMessage );

cy.get(@submitButton).click();
```
nice, less code, don't have to import and export, it's part of global `cy` (although hope it's not a performance/scaling issue storing a lot of aliases in `cy`)

Reply:
Guess not :-(
```
! Uncaught CypressError
Cannot call cy.get() outside a running test.
This usually happens when you accidentally write commands outside an it(...) test.
If that is the case, just move these commands inside an it(...) test.
```
So can't seem to use in a global scenario, just per test.  Seems like aliases are typically used in the `before()` and `after()` hooks of the tests.